src/bridge/ui_commands.rs

```diff
@@ -56,6 +65,7 @@ impl UiCommand {
                         .expect("Mouse Input Failed");
                 }
             }
+            #[cfg(not(feature = "glfw"))]
```

Comment:
This is pretty important to not include.

Reply:
I know, for now, the branch is mostly prove the theory, I will try to implement the rest of them in spare time.

Reply:
https://github.com/Kethku/neovide/pull/364/commits/a97fb234ea7eeefc5c8f0f3b89f54b8f227d6da4

Reply:
https://github.com/Kethku/neovide/pull/364/commits/2c3cde00d3fe4c9d54d8ef75a6afe0cad154f7c8